pkgs/by-name/ma/manager-io-server/package.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, stdenvNoCC
```

Comment:
The CI failed because it checks that each file is formatted correctly (with nixfmt)